samples/excel/85-preview-apis/range-get-range-edge.yaml

```
@@ -0,0 +1,229 @@
+order: 8
+id: excel-get-range-edge
+name: Select used range edge
+description: This sample shows two ways to select the edge of the used range, based on the currently selected range.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+    content: |
+      $("#setup").click(() => tryCatch(setup));
+      $("#select-E9").click(() => tryCatch(selectE9));
+      $("#select-D8-E9").click(() => tryCatch(selectD8E9));
+      $("#get-range-edge-left").click(() => tryCatch(getRangeEdgeLeft));
+      $("#get-range-edge-up").click(() => tryCatch(getRangeEdgeUp));
+      $("#get-extended-range-right").click(() => tryCatch(getExtendedRangeRight));
+      $("#get-extended-range-down").click(() => tryCatch(getExtendedRangeDown));
+
+      async function getRangeEdgeLeft() {
+        await Excel.run(async (context) => {
+          // Get the selected range.
+          const range = context.workbook.getSelectedRange();
+
+          // Get the active cell in the workbook.
+          const activeCell = context.workbook.getActiveCell();
+
+          // Get the cell at the furthest edge of the current used range, in the "Left" direction.
+          // This action matches the result of using the Control+Arrow key keyboard shortcut while a range is selected.
+          const rangeEdge = range.getRangeEdge(
+            "Left", // Define the direction as a string.
+            activeCell // If the selected range contains more than one cell, the active cell must be defined.
+          );
+
+          // Select the edge of the range.
+          rangeEdge.select();
+
+          await context.sync();
+        });
+      }
+
+      async function getRangeEdgeUp() {
+        await Excel.run(async (context) => {
+          // Get the selected range.
+          const range = context.workbook.getSelectedRange();
+
+          // Define the direction with the `KeyboardDirection` enum.
+          const direction = Excel.KeyboardDirection.up;
+
+          // Get the active cell in the workbook.
+          const activeCell = context.workbook.getActiveCell();
+
+          // Get the cell at the furthest edge of the current used range, in the "up" direction.
+          // This action matches the result of using the Control+Arrow key keyboard shortcut while a range is selected.
```

Same comments as the previous call.
```suggestion
          // This method acts like the Ctrl+Arrow key keyboard shortcut while a range is selected.
```